hopr/hopr-lib/src/lib.rs

```diff
@@ -1150,6 +1150,11 @@ impl Hopr {
         Ok(self.chain_api.ticket_price().await?)
     }
 
+    /// Get ticket winning probability
+    pub async fn get_ticket_probability(&self) -> errors::Result<f64> {
+        Ok(hopr_internal_types::tickets::WINNING_PROB)
+    }
+
```

Comment:
If this call cannot produce an error, why would it have an API that can produce an error state?

Reply:
When we will use an oracle for the winning prob, the API will be able to produce an error state. Structure is already there so that it won't change when moving to using an oracle.